index.html

```diff
@@ -277,37 +277,83 @@ <h6 class="link-item">
+                    <p class="code-title">CSS</p>
+                  </div>
                 </li>
               </ul>
             </div>
             <div class="skill-container">
-                <div class="skill-title">
-                  <ul>
-                    <li><a href="#" class="link-skill">Frameworks</a></li>
-                  </ul>
-                </div>
-                <div class="vector-icon"></div>
+              <div class="skill-title">
+                <ul>
+                  <li><a href="#" class="link-skill">Frameworks</a></li>
+                </ul>
               </div>
-              <div class="skill-container">
-                <div class="skill-title">
-                  <ul>
-                    <li><a href="#" class="link-skill">Skills</a></li>
-                  </ul>
-                </div>
-                <div class="vector-icon"></div>
+              <div class="vector-icon"></div>
+            </div>
+            <div class="skill-container">
+              <div class="skill-title">
+                <ul>
+                  <li><a href="#" class="link-skill">Skills</a></li>
+                </ul>
               </div>
+              <div class="vector-icon"></div>
+            </div>
           </div>
-          </div>
+          <section id="contact-section">
+            <div class="content contact-container">
+              <div class="contact-title-container">
+                <h2 class="contact-title">Contact me</h2>
+              </div>
+              <div class="contact-description-container">
+                <p class="contact-description">
+                  If you have an application you are interedte in developing, a
+                  feature that you need built or a project that need coding. I'd
+                  love to help with it
+                </p>
+              </div>
+              <div class="form-container">
+                <form class="contact-form">
+<<<<<<< HEAD
+                  <input
+                    type="text"
+                    placeholder="Name"
+                    class="input-client-info"
+                  />
+                  <input
+                    type="email"
+                    name="email"
+                    placeholder="example@gmail.com"
+                    class="input-client-info"
+                  />
+                  <textarea
+                    name="message"
+                    id="message"
+                    placeholder="right your message here"
+                    class="client-message"
+                  >
+=======
+                  <input type="text" placeholder="Name" class="input-client-info" minlength="10" maxlength="30" required>
+                  <input type="email" name="email" placeholder="example@gmail.com" class="input-client-info" required>
+                  <textarea name="message" id="message" placeholder="right your message here" class="client-message" minlength="10" maxlength="500" required>
```

Comment:
- [ ] Once the merge problems are fixed, you should use those inputs, considering they match all the project's requirements.
- [ ] [OPTIONAL] You should consider lowering the `minlength` required for the name input, for instance, my name would be too short, and some people might not want to use their fullnames in here!